### ISSUE TITLE
How do I specify AWS credentials for S3 URLs with the Gradle plugin?

### ISSUE THREAD

**Issue by ratkins** [DiUS/pact-jvm#593]
The README for the gradle plugin mentions it's possible to [verify pact files from an S3 bucket](https://github.com/DiUS/pact-jvm/tree/master/pact-jvm-provider-gradle#verifying-pact-files-from-a-s3-bucket-version-3322417). [ELI5](https://www.reddit.com/r/explainlikeimfive/) how I specify the AWS secret key and/or access key? (Preferably with an example!)

**Comment by uglyog** [comment]
By default, the AWS SDK will look for credentials in `~/.aws/credentials`. You can just create this file.

```

**Comment by ratkins** [comment]
In the end, we solved this by adding `AWS_ACCESS_KEY_ID` and `AWS_SECRET_KEY` to the environment of the shell script which starts up our server and runs the Pact tests.
